README.md

```diff
@@ -130,6 +130,24 @@ sqlContext.sql("CREATE TEMPORARY TABLE students_table USING com.stratio.provider
 sqlContext.sql("SELECT * FROM students_table").collect()
 
 ```
+
+### R API
+
```

Comment:
Hi, first of all, thank you very much!

We don't know what environment are you using (Rstudio, sparkR shell, etc) and we need to show how to set the environment in the documentation, all the step for run the example correctly.